#! /bin/sh /usr/share/dpatch/dpatch-run
## sqlite3-support.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: SQLite3 support. Fixes upstream http://freepbx.org/trac/ticket/1791

@DPATCH@
diff -urNad freepbx-module-findmefollow~/install.sql freepbx-module-findmefollow/install.sql
--- freepbx-module-findmefollow~/install.sql	2006-10-15 11:11:20.000000000 +0200
+++ freepbx-module-findmefollow/install.sql	2007-02-18 11:39:26.000000000 +0200
@@ -1,3 +1,3 @@
 
-CREATE TABLE IF NOT EXISTS `findmefollow` ( `grpnum` BIGINT( 11 ) NOT NULL , `strategy` VARCHAR( 50 ) NOT NULL , `grptime` SMALLINT NOT NULL , `grppre` VARCHAR( 100 ) NULL , `grplist` VARCHAR( 255 ) NOT NULL , `annmsg` VARCHAR( 255 ) NULL , `postdest` VARCHAR( 255 ) NULL , `dring` VARCHAR ( 50 ) NULL , remotealert VARCHAR ( 80 ), needsconf VARCHAR ( 10 ), toolate VARCHAR ( 80 ), pre_ring SMALLINT NOT NULL DEFAULT 0, PRIMARY KEY  (`grpnum`) ) TYPE = MYISAM ;
+CREATE TABLE IF NOT EXISTS `findmefollow` ( `grpnum` BIGINT( 11 ) NOT NULL , `strategy` VARCHAR( 50 ) NOT NULL , `grptime` SMALLINT NOT NULL , `grppre` VARCHAR( 100 ) NULL , `grplist` VARCHAR( 255 ) NOT NULL , `annmsg` VARCHAR( 255 ) NULL , `postdest` VARCHAR( 255 ) NULL , `dring` VARCHAR ( 50 ) NULL , remotealert VARCHAR ( 80 ), needsconf VARCHAR ( 10 ), toolate VARCHAR ( 80 ), pre_ring SMALLINT NOT NULL DEFAULT 0, PRIMARY KEY  (`grpnum`) );
 
